docs/ql-libraries/dataflow/dataflow.md

```diff
@@ -98,10 +98,6 @@ Recommendations:
   also work, but the upside of `use-use` steps is that sources defined in terms
   of variable reads just work out of the box. It also makes certain
   barrier-implementations simpler.
-* A predicate `DataFlowCallable Node::getEnclosingCallable()` is required, and in
```

Comment:
With this deletion I think the document no longer mentions that `DataFlowCallable Node::getEnclosingCallable()` needs to exist. The doc really ought to say something about how a "scope" aka "enclosing callable" is required for all nodes.